_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the ability to:
+
+- Call OpenSearch indexes and manipulate using the opensearch-py-ml [DataFrame](https://opensearch-project.github.io/opensearch-py-ml/reference/dataframe.html) APIs. The opensearch-py-ml DataFrame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
+-  Upload NLP [SentenceTransformers](https://www.sbert.net/) models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/).
+- Train and tune SentenceTransformer models with ML Common and synthetic queries.
+
+## Prerequisites 
+
+To use `opensearch-py-ml`, install the [OpenSearch Python client]({{site.url}}{{site.baseurl}}/clients/python#setup). The Python client allows OpenSearch to use the Python syntax required to run Data Frames in `opensearch-py-ml`.
+
+## Install `opensearch-py-ml`
+
+To add the client to your project install it using [pip](https://pip.pypa.io/):
+
+```bash
+pip install opensearch-py-ml
+```
+
+Then, import the client into OpenSearch like any other module:
```

Comment:
```suggestion
Then import the client into OpenSearch like any other module:
```

Reply:
Skip the comma after single-word adverbs of time at the beginning of a sentence, such as afterward, then, later, or subsequently.